src/twisted/internet/defer.py

```diff
@@ -159,6 +165,65 @@ def maybeDeferred(f, *args, **kw):
         return succeed(result)
 
 
+def unbox(deferred, raises=True, passthrough=True):
```

Comment:
My intuition here is that:

```python
def unbox(deferred, raises=True, consumeErrors=True):
    ...
```

Is the "right" way forward, but I'm not as versed in what is normal in Twisted land.